src/api/app/assets/stylesheets/webui/patchinfo.scss

```
@@ -16,25 +16,25 @@
 
 .patchinfo {
   .badge {
-    &.badge-rating, &.badge-category, &.badge-blocked {
+    &.bg-rating, &.bg-category, &.bg-blocked {
```

```suggestion
    &.badge-rating, &.badge-category, &.badge-blocked {
```
I would leave this three `.badge-` classes as they are already defined:
- They don't only modify the background (with the exception of `.badge-blocked`).
- They are only applied to badges.
- They could get confused with the default `.bg-` Bootstrap 5 classes.